test/cbmc/proofs/Malloc_FreeRTOS/Malloc_FreeRTOS_harness.c

```diff
@@ -29,6 +29,8 @@
 /*  FreeRTOS includes for OTA library. */
 #include "ota_os_freertos.h"
 
+#include <stdlib.h>
```

Comment:
NIT: should we not include such standard libs before the headers of our own libraries as a rule of FreeRTOS?

Reply:
Happy to adjust this if there is such a rule, I wasn't aware.